build/Makefile

```diff
@@ -64,7 +64,7 @@ KIND_PROFILE ?= agones
 KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 
 # Game Server image to use while doing end-to-end tests
-GS_TEST_IMAGE ?= us-docker.pkg.dev/agones-images/examples/simple-game-server:0.32
+GS_TEST_IMAGE ?= us-central1-docker.pkg.dev/agones-ashutoshnsingh/example/simple-game-server:0.33-dev-linux-amd64
```

Comment:
I expect this won't work, since the `agones-images` project doesn't have access to your artifact registry.

You could make it public if yo u desired though 😄 at least temporarily.

Reply:
Thanks a lot @markmandel. It worked perfectly.